python/ray/data/dataset.py

```diff
@@ -1772,7 +1788,9 @@ def unique(self, column: str) -> List[Any]:
 
     @ConsumptionAPI
     def aggregate(self, *aggs: AggregateFn) -> Union[Any, Dict[str, Any]]:
-        """Aggregate the entire dataset as one group.
+        """Aggregate values using one or more functions.
+
+        Use this method to compute metrics like the mean of a column.
 
         Examples:
             >>> import ray
```

Comment:
I think showing a custom aggregation would be more meaningful since mean and max have built in methods already.